src/main/java/org/spongepowered/api/block/entity/BlockEntity.java

```diff
@@ -67,6 +67,27 @@ public interface BlockEntity extends SerializableDataHolder.Mutable, Locatable {
      */
     void remove();
 
+    /**
+     * Returns whether this block entity can tick.
+     *
+     * @return True if this block entity can tick
+     */
+    boolean canTick();
+
+    /**
+     * Returns whether this block entity is ticking.
+     *
+     * @return True if this block entity is ticking
+     */
+    boolean isTicking();
+
+    /**
+     * Sets if this block entity will naturally tick.
+     *
+     * @param ticking The ticking state
+     */
+    void setTicking(boolean ticking);
```

Comment:
Can we just clarify in some more detail if we can actually override whether a block entity can tick? And if a block entity normally doesn't tick, that this would do nothing? Perhaps having the return value `boolean` signify the success or failure of being able to set this block entity to tick/not?

Reply:
The point here is that if block entity `canTick()` and not `isRemoved()` then we can change it's ticking state (in impl it's simply cancelling tick if this state is false). Impl currently throws exceptions if these checks are not passed (probably worth mentioned this in api). So which way is better: throw if dev didn't checked these conditions or have `boolean` result?

Reply:
I think having a boolean result (imo) would be ideal. It still allows passive setting, and then developers can elect to provide feedback for the failure mode.

Reply:
Done